edx_proctoring/backends/rest.py

```diff
@@ -349,6 +353,9 @@ def get_onboarding_profile_info(self, course_id, **kwargs):
         data = response.json()
         return data
 
+    def translate_filter_status(self, filter_status):
```

Comment:
I moved the translation of statuses here, should we do that for other backend specific translations?